programs/gauge/src/state.rs

```diff
@@ -395,19 +387,22 @@ impl GaugeVoter {
         return Err(VotingEpochNotFound.into());
     }
 
-    // should return if it not pump
+    /// voting epoch start from 1
+    /// max index that we can store voting epoch is MAX_EPOCH_PER_GAUGE
+    /// so index_for_latest_voting_epoch = latest_voting_epoch % MAX_EPOCH_PER_GAUGE - 1 if latest_voting_epoch != MAX_EPOCH_PER_GAUGE
+    /// else index_for_latest_voting_epoch = MAX_EPOCH_PER_GAUGE - 1
```

Comment:
we should remove comment here? It is incorrect

Reply:
removed